### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by claudion3**

### CODE REVIEW

File: databases/week2/school_database/schoolDB.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+begins datetime,
+ends datetime
+);
+-- creating student table
+create table student(
+id int unsigned primary key auto_increment,
+studentname varchar(255) not null,
+email varchar(255) not null unique,
+phone int unique,
+class_id int unsigned not null,
```
Comment (babakbandpey) [inline databases/week2/school_database/schoolDB.sql:16]: I am sorry for repeating myself, but this class_id is not belong to this table.
Can an student exist without being a part of a class?
Yes the student can exist in the system without being enrolled into any classes.

I believe that I have explained that you need to create a relational model which can accommodate many classes for one student and many students for one class.

**Review by babakbandpey — CHANGES_REQUESTED** [review]
Hello,
Please create a table which is a relation between the classes and the students.
I have commented on it.
Thanks.

**Review by babakbandpey — CHANGES_REQUESTED** [review]
Hello again,

You have removed the school schema and most of my comments revolved around perfecting that model. So I do not see why we had these reviews.
Please remove the PDF file and the the school.mwb.bak and I will partially approve your PR.

Thank you.

**Comment by claudion3** [comment]
hello Babak, 
you are not clear when you give the reviews i don't know if i'm only one who don't get what you means ex: this what you wrote on previous reviews (Please create a table which is a relation between the classes and the students.) and i did you can also see it on that pdf but i can't upload the data it get encrypted when i upload mwb file. and i was checking what other did to see if i'm missing something and i saw also they added pdf.

so, what exactly you want me to change?

**Pull request merged**